tutorial chapter 01 build fails

see https://cerebral.github.io/tutorial/01/

There is errors during build of chapter 1: 
```
Module not found: Error: Cannot resolve 'file' or 'directory' ./components/App in /home/travis/build/cerebral/cerebral/packages/tutorial/src
```

--- a/packages/tutorial/scripts/prepare.js
+++ b/packages/tutorial/scripts/prepare.js
@@ -1,4 +1,4 @@
-/* global rm, ls, cp, sed */
+/* global rm, mkdir, ls, cp, sed */
 var path = require('path')
 var paths = require('./paths')
 
@@ -17,6 +17,8 @@ function prepare (part) {
         rm('-rf', path.join(paths.src, file))
       }
     })
+  } else {
+    mkdir(paths.src)
   }
 
   ls(path.join(paths.parts, part, 'src')).forEach(function (file) {

